core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -980,7 +980,9 @@ public enum BuiltInMethod {
       ImmutableBitSet.class),
   FUNCTIONAL_DEPENDENCY_DETERMINANTS(FunctionalDependency.class, "determinants",
       ImmutableBitSet.class),
-  FUNCTIONAL_DEPENDENCY_GET_FDS(FunctionalDependency.class, "getFDs");
+  FUNCTIONAL_DEPENDENCY_GET_FDS(FunctionalDependency.class, "getFDs"),
+  AGE(SqlFunctions.class, "age", long.class, long.class),
+  AGE_ONE_PARAM(SqlFunctions.class, "age", long.class);
```

Comment:
I'm wondering if we could refer to the implementation of SqlSubstringFunction, since SUBSTRING can also accept two or three parameters. Is it possible to avoid declaring two functions based on the number of input parameters here?